core/client/app/components/gh-cm-editor.js

```diff
@@ -18,21 +18,14 @@ export default Ember.Component.extend({
 
     didInsertElement: function () {
         var options = this.getProperties('lineNumbers', 'indentUnit', 'mode', 'theme'),
-            self = this,
-            editor;
-        editor = new CodeMirror(this.get('element'), options);
+            editor = new CodeMirror(this.get('element'), options);
+
         editor.getDoc().setValue(this.get('value'));
 
         // events
-        editor.on('focus', function () {
-            self.set('isFocused', true);
-        });
-        editor.on('blur', function () {
-            self.set('isFocused', false);
-        });
-        editor.on('change', function () {
-            self.set('value', editor.getDoc().getValue());
-        });
+        editor.on('focus', Ember.run.bind(this, 'set', 'isFocused', true));
+        editor.on('blur', Ember.run.bind(this, 'set', 'isFocused', false));
+        editor.on('change', Ember.run.bind(this, 'set', 'value', editor.getDoc().getValue()));
```

Comment:
Does this not just evaluate `editor.getDoc().getValue()` once, rather than every time the change happens?